src/data.py

```diff
@@ -204,7 +204,12 @@ def parse_static_eateries(static_json):
         location=eatery.get('location', ''),
         name=eatery.get('name', ''),
         name_short=eatery.get('nameshort', ''),
-        operating_hours=parse_static_op_hours(eatery['operatingHours'], new_id, dining_items),
+        operating_hours=parse_static_op_hours(
+            eatery['operatingHours'],
```

Comment:
jw is `operatingHours` camel case because it's a constant?

Reply:
its camel because it comes from a json file, which I think is camel standard